addons/resource/models/resource_calendar.py

```diff
@@ -195,43 +277,23 @@ def _get_hours_per_day(self, attendances):
 
     def switch_calendar_type(self):
```

Comment:
Depending on how are handled the changes for attendances, this method will have to change accordingly, but I'm not sure if the current changes are necessary :thinking: